src/Layout/redist/targets/OverlaySdkOnLKG.targets

```diff
@@ -25,10 +25,13 @@
       </TemplatesFolderPath>
     </ItemGroup>
     <Error Text="SDK Stage 0 has more than one folder with templates: @(TemplatesFolderPath->'%(Identity)'). Please delete all but one and rebuild" Condition="@(TemplatesFolderPath->Count()) > 1"></Error>
+    <PropertyGroup>
+      <TemplateDestinationPath>$(RedistLayoutPath)templates\$([System.IO.Path]::GetFileName(@(TemplatesFolderPath)))</TemplateDestinationPath>
```

Comment:
squashing the Item list into a single property, since the Error before validated that we only have a single item